stock_available_to_promise_release/migrations/14.0.2.5.0/post-migrate.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(cr, version):
+    cr.execute(
+        """
+        UPDATE
+            stock_picking
+        SET
+            release_policy = sub.move_type
+        FROM (
+            SELECT
+                p.id,
+                g.move_type
+            FROM
+                stock_picking as p
+            JOIN
+                stock_move as m
+            ON
+                m.picking_id = p.id
+            JOIN
+                procurement_group as g
+            ON
+                m.group_id = g.id
+        ) as sub
+        WHERE
+            stock_picking.id = sub.id
```

Comment:
```suggestion
            release_policy = procurement_group.move_type
        FROM
                procurement_group
        WHERE
            stock_picking.group_id = procurement_group.id
```

Reply:
oh yes indeed the group_id is stored. Thxs